cluster-soak/src/main/scala/akka/kubernetes/soak/PingPongActor.scala

```diff
@@ -42,17 +42,25 @@ object Tests {
   type ResponseTimeNanos = Long
 }
 
-case class TestResult(notResponded: Set[Target], responses: List[(Target, ResponseTimeNanos)])
+case class TestResult(notResponded: Set[Target], responses: List[(Target, ResponseTimeNanos)]) {
+  override def toString: Target = s"TestResult(notResponded: $notResponded)"
+}
 
 case class GetTestResults(resetFailures: Boolean = true)
+
 case class TestResults(testsRun: Long,
                        testsFailed: Long,
                        lastResult: TestResult,
                        recentFailures: List[TestResult],
                        memberDownedEvents: Long,
-                       memberUnreachableEvents: Long)
+                       memberUnreachableEvents: Long,
+                       joiningTime: Long) {
+
+  override def toString =
+    s"TestResults(testsRun: $testsRun, testsFailed: $testsFailed, lastResult: $lastResult, recentFailed: $recentFailures, downed: $memberDownedEvents, unreachable: $memberUnreachableEvents, joiningTime: $joiningTime)"
```

Comment:
do we need a custom `toString` here?

Reply:
I preferred adding in the names of some of the fields when looking at results as i always forgot which number was which